app/src/main/res/values/dimens.xml

```diff
@@ -10,6 +10,7 @@
 
     <!-- Search -->
     <dimen name="search_speaker_image_size">40dp</dimen>
+    <dimen name="search_result_top_margin">18dp</dimen>
```

Comment:
🙋 
Why is this `18dp`? the item on the top has `6dp` margin?

Reply:
Yes, The item has 6dp top and bottom margin.
https://github.com/DroidKaigi/conference-app-2018/blob/master/app/src/main/res/layout/item_speech_session.xml

Reply:
Understood.
I think the item layout should not have `marginTop`...
But this layout is used in some places, so your fix seems enough.
I'll create another issue to remove `marginTop` in the item layout 👍 

Thanks!

Reply:
Created the issue!
https://github.com/DroidKaigi/conference-app-2018/issues/172